tests/pl/test_render_shapes.py

```diff
@@ -101,6 +102,26 @@ def test_plot_can_color_from_geodataframe(self, sdata_blobs: SpatialData):
     def test_plot_can_scale_shapes(self, sdata_blobs: SpatialData):
         sdata_blobs.pl.render_shapes(elements="blobs_circles", scale=0.5).pl.show()
 
+    def test_plot_can_plot_with_group_arg(self, sdata_blobs: SpatialData):
+        sdata_blobs.shapes["blobs_polygons"]["cluster"] = "c1"
+        sdata_blobs.shapes["blobs_polygons"].iloc[3:5, 1] = "c2"
+        sdata_blobs.shapes["blobs_polygons"]["cluster"] = sdata_blobs.shapes["blobs_polygons"]["cluster"].astype(
+            "category"
+        )
+
+        sdata_blobs.pl.render_shapes("blobs_polygons", color="cluster", groups=["c1"]).pl.show()
+
+    def test_plot_can_plot_with_group_and_palette(self, sdata_blobs: SpatialData):
+        sdata_blobs.shapes["blobs_polygons"]["cluster"] = "c1"
+        sdata_blobs.shapes["blobs_polygons"].iloc[3:5, 1] = "c2"
+        sdata_blobs.shapes["blobs_polygons"]["cluster"] = sdata_blobs.shapes["blobs_polygons"]["cluster"].astype(
+            "category"
+        )
+
+        sdata_blobs.pl.render_shapes(
+            "blobs_polygons", color="cluster", groups=["c2", "c1"], palette=ListedColormap(["green", "yellow"])
```

Comment:
Please take care of the ListedColormap conversion inside the code, it's unintuitive to have the user feed it in.